### ISSUE TITLE
LineChart accessibility issue for the data tooltip - Cannot hover over it unless focused with keyboard

### ISSUE THREAD

**Issue by gencay** [microsoft/fluentui#21371]
### Environment Information

- **Package version(s)**: 4.18.5
- **Browser and OS versions**: Windows 11 / Edge Version 97.0.1072.62 (Official build) (64-bit)

### Describe the issue:

User should be able to move over to the tooltip when hovering a point in line chart

![image](https://user-images.githubusercontent.com/1168485/150454888-ce9f083a-fa2f-4c47-86c9-56a3eaf3fb99.png)

### Please provide a reproduction of the issue in a codepen:

https://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/charting/dist/index.html#/examples/linechart

#### Actual behavior:

Unable to move over the tooltip with mouse when a line chart point is hovered.

#### Expected behavior:

When mouse leaves the data point, the tooltip should remain visible for a few seconds to enable user to be able to hover over it. Otherwise, it immediately gets hidden without being able to hover over the tooltip.
